tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/AnonymousObjectTest.java

```diff
@@ -169,6 +169,11 @@ public void testCodeAnalyzerRunningOnAnonymousObjectsForDeprecatedFunctionAnnota
         BAssertUtil.validateWarning(compileResult, 1, "usage of construct 'Test' is deprecated", 287, 25);
     }
 
+    @Test
+    public void testLocalVariablesFromFields() {
```

Comment:
Check the comment in the test bal file. This is more an object constructor test.

Reply:
Changed